server/plugin/mm_34646_token_refresh.go

```diff
@@ -0,0 +1,125 @@
+package plugin
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/pkg/errors"
+)
+
+const pageSize = 100
+const delayBetweenPages = 5 * time.Minute
```

Comment:
what do we gain by having such delay? Would it be better to introduce some (smaller) delay for after each (key) reset?

Reply:
Yeah, I also don't know what the server considers "bad" request policy - blasts coming at larger interval vs. a constant trickle. 0/5 as well, unless we get more context.

Reply:
Based on these comments, https://github.com/mattermost/mattermost-plugin-github/pull/472#discussion_r693777591 and https://github.com/mattermost/mattermost-plugin-github/pull/472#discussion_r693778173 I'll do a 1-time delay of say 5 minutes, and then a slow trickle, 1 at a time. cc @hanzei